### ISSUE TITLE
The nvda to randomly read the emails in outlook's inbox

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9389]
### Steps to reproduce:
Open the outlook go to the inbox and work a little opening the messages, closing the messages.
After some time while walking with the arrows above or below the nvda does not read the messages in the inbox.
If I open the message it is read, if I close the message with esc the message in the input box is read, but if I press down arrow the next message is not read.
### Actual behavior:
randomly to read the messages in the inbox while walking with arrows above and below.
### Expected behavior:
Nvda should be able to continue working with Outlook for long periods without reading the messages in the inbox while navigating through the messages with the above and below arrows.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
2018.4.1
#### Windows version:
10 1803
#### Name and version of other software in use when reproducing the issue:
Microsoft® Outlook® 2016 MSO (16.0.4266.1001) 64 bits 

#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
log:
[inbox.txt](https://github.com/nvaccess/nvda/files/2988779/inbox.txt)

**Comment by DrSooom** [comment]
@fernando-jose-silva: Try to disable the mail preview (in Outlook 2010: View » Layout » Reading area (or so) » Off) and maximize the window with Win+ArrowUp. Any changes?

**Comment by fernando-jose-silva** [comment]
Thanks a lot, I disabled the reading mode, the outlook was already maximized.
For now everything works fine.
If you can wait until morning, as the lack of reading is random, if today work all day long well I believe we can close.

**Comment by fernando-jose-silva** [comment]
@DrSooom 
Thank you very much, with your guidelines everything worked.
Strange in my opinion should work with the display of the messages in the reading panel.
Thank you very much, we can close the call.

**Comment by ehollig** [comment]
Closing as requested.

**Comment by fernando-jose-silva** [comment]
If we could return to investigate this problem a little more or less 1 week ago I have this problem again even with the reading panel disabled, there may be some other question that is causing the items not to be read in the box. input.
[outlook.txt](https://github.com/nvaccess/nvda/files/3044120/outlook.txt)
